### DISCUSSION TITLE
Welcome to firefly Discussions!

### DISCUSSION THREAD

**Discussion by Safe3** [Safe3/firefly#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Amaan1532** [comment]
Hlo sir
My name is amaan malik and i study at rohilkhand university bareilly and i have a diploma of cs python can you provide me any project sir please
